doc/user/content/transform-data/patterns/durable-subscriptions.md

```diff
@@ -1,42 +1,45 @@
 ---
-title: "Time travel queries"
-description: "How to use history retention to enable time travel queries in Materialize"
+title: "Durable subscription (Retain History)"
+description: "How to configure history retention to set up durable subscriptions in Materialize"
 menu:
   main:
     parent: 'sql-patterns'
-    name: "Time travel queries"
+aliases:
+  - /transform-data/patterns/time-travel-queries/
+  - /transform-data/patterns/time-travel-queries/#history-retention-period
 ---
 
-{{< private-preview />}}
+You can configure an object's [history retention
+period](#history-retention-period) for durable subscriptions.
 
-By default, all user-defined sources, tables, materialized views, and indexes
-keep track of the most recent version of their underlying data. Some use cases
-require accessing past versions of this data at specific points in time
-(i.e. time traveling). To allow time travel queries, you can configure these
-objects to **retain history**.
+{{< private-preview />}}
 
-When set to a value different than the default (i.e. no history retention), the
-history retention period of an object determines the duration for which
-historical versions of its underlying data are available for querying.
+## Durable subscriptions
 
-The common use cases for time traveling in Materialize are:
+Because [`SUBSCRIBE`](/sql/subscribe/) requests happen over the network, these
+connections might get disrupted for both expected and unexpected reasons. To
+avoid the need for re-snapshotting the data, you can:
 
-* Lossless, **durable subscriptions** to your changing results. See the
-  [durable subscriptions section below](#durable-subscriptions)
-  for details on how to create this type of subscription.
+- Adjust the [history retention period](#history-retention-period) for the
+  objects that a subscription depends on, and then
 
-* Accessing a past version of results at a specific point in time.
+- Use [`AS OF`](/sql/subscribe#as-of) to pick up where you left off on
+  connection drops.
```

Comment:
```suggestion
- [Access past versions of this data](#enabling-durable-subscriptions-in-your-application)
  at specific points in time to pick up data processing where you left off.
```

Reply:
Might be more user-friendly to structure this as a guide with two steps and/or explicitly point to each section instead of using implicit links.